src/translations/pages/contact.ts

```diff
@@ -974,4 +974,38 @@ export const Contact = {
       },
     },
   },
+  success: {
+    backButton: _(
```

Comment:
Does  back to the ( 'hovedsiden' 'main page', 'hovedsida') sound better?

Reply:
Not sure, but updated